exonum/src/node/consensus.rs

```diff
@@ -384,6 +384,14 @@ where
 
             // Change lock
             if self.state.has_majority_prevotes(round, propose_hash) {
+
+                // Put our consensus messages to the cache. This is for validators only.
+                if self.state.is_validator() {
+                    self.blockchain
+                        .save_messages(self.state.prepare_consensus_messages().into_iter())
```

Comment:
You can use `IntoIterator` instead of `Iterator` in `save_messages` to avoid `into_iter()` call.

Reply:
Done